dockerfile

```diff
@@ -0,0 +1,16 @@
+# Use the latest version of the Rust base image
```

Comment:
I find a bit confusing to have one dockerfile here and others in https://github.com/eclipse-kuksa/kuksa-databroker/blob/main/scripts/Dockerfile

What is the difference between them. Should better be described somewhere.